src/agentscope/service/web_search/web_digest.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+"""parsing and digesting the web pages"""
+from typing import Optional, Union
+import requests
+from loguru import logger
+
+from agentscope.service.service_response import ServiceResponse
+from agentscope.service.service_status import ServiceExecStatus
+from agentscope.models.model import ModelWrapperBase
+from agentscope.prompt import PromptEngine, PromptType
+
+
+DEFAULT_SYS_PROMPT = (
+    "You're a web page analyser. You job is to extract important"
+    "and useful information from html or webpage description.\n"
+)
+
+DEFAULT_SUMMARY_PROMPT = (
+    "You're a web page analyser. You job is to summarize the "
+    "content of a webpage.\n"
+)
+
+DEFAULT_SPLIT_LEVEL = [
+    ("h1", "h1"),
+    ("h2", "h1"),
+]
+
+
+def webpage_digest(
```

Comment:
Extract a `load_webpage` method from this method.
It focuses on getting web pages and cleaning them without calling LLM.
Agents can use `load_webpage` to obtain the cleaned web page.

Reply:
Done